app/services/trainees/set_slug_sent_at.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Trainees
+  class SetSlugSentAt
+    include ServicePattern
+
+    def initialize(trainee:)
+      @trainee = trainee
+    end
+
+    def call
+      trainee.slug_sent_to_dqt_at ||= Time.zone.now
```

Comment:
Shouldn't it set it every time regardless? Feels odd that we only record once.

Reply:
@defong originally I was going to just make this a boolean but then I thought that we would normally record the time that a thing happened. So this isn't a direct requirement and therefore it's totally open to debate! My thinking was that we are most likely to care about when we first synched slugs with DQT (e.g. if you ever needed to know whether DQT had the slug at a particular date in the past).

Reply:
Oh right I was thinking we have like the timeline and the audit, hence the oddity.

I think it maybe better to have it set every time, so we can say we set it on the firstly and subsequently on theses dates based on audit and last recorded date.

WDYT?